content/blog/2019-08-15-new-react-devtools.md

```diff
@@ -37,14 +37,14 @@ If you use the standalone shell (e.g. in React Native or Safari), you can instal
 npm install -g react-devtools@^4
 ```
 
-## Where did all of the DOM elements go?
+## Where did all of the DOM elements go? {#where-did-all-of-the-dom-elements-go}
 
 The new DevTools provides a way to filter components from the tree to make it easier to navigate deeply nested hierarchies.
 Host nodes (e.g. HTML `<div>`, React Native `<View>`) are *hidden by default*, but this filter can be disabled:
 
 ![DevTools component filters](../images/blog/devtools-component-filters.gif)
 
-## How do I get the old version back?
+## How do I get the old version back? {#how-do-i-get-the-old-version-back}
```

Comment:
`## How do I get the old version back? {#how-do-i-get-the-old-version-back}` => `## How do I get the old version back?`